terms/demographics/Braak.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.Braak-0.0.1",
+  "type": "string",
+  "description": "Braak Stage - a semiquantitative measure of serverity of neurofibrillary tangle pathology",
+  "anyOf": [
+    {
+      "const": 0,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    },
+    {
+      "const": 1,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    },
+    {
+      "const": 2,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    },
+    {
+      "const": 3,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    },
+    {
+      "const": 4,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    },
+    {
+      "const": 5,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    },
+    {
+      "const": 6,
+      "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894536/"
+    }
+  ]
+}
```

Comment:
I agree that this would be preferable if we support min/max -- I don't know off the top of my head, but we can ask John

Reply:
We do not have support for minimum/maximum, yet, but there is at least one other term that was merged to production with minimum/maximum. Since we already use boolean, which is also not supported, yet, I think it would be fine to treat these as numeric with min/max and fix later, if needed (same thing we will have to do if boolean doesn't get supported).

Reply:
Since this is neuropathology, we should move this to clinical instead of demographics